services/catalog/src/simcore_service_catalog/core/application.py

```diff
@@ -31,24 +32,25 @@
 
 def init_app(settings: Optional[AppSettings] = None) -> FastAPI:
     if settings is None:
-        settings = AppSettings.create_default()
+        settings = AppSettings.create_from_envs()
 
-    logging.basicConfig(level=settings.loglevel)
-    logging.root.setLevel(settings.loglevel)
+    logging.basicConfig(level=settings.CATALOG_LOG_LEVEL.value)
+    logging.root.setLevel(settings.CATALOG_LOG_LEVEL.value)
+    logger.debug(settings.json(indent=2))
 
     app = FastAPI(
-        debug=settings.debug,
-        title="Components Catalog Service",
-        description="Manages and maintains a **catalog** of all published components (e.g. macro-algorithms, scripts, etc)",
+        debug=settings.SC_BOOT_MODE
+        in [BootModeEnum.DEBUG, BootModeEnum.DEVELOPMENT, BootModeEnum.LOCAL],
+        title=project_name,
```

Comment:
MINOR: all the ``_meta`` we should slowly turn into upper case since they are constants